tiledb/common/logger.cc

```diff
@@ -129,38 +178,38 @@ Logger& global_logger() {
 
 /** Logs an error. */
 void LOG_TRACE(const std::string& msg) {
-  global_logger().trace(msg.c_str());
+  global_logger().trace(msg);
```

Comment:
Should we make the stringstream version of these too?

Reply:
Good though, done.